inlong-sdk/sort-sdk/pom.xml

```diff
@@ -27,7 +27,7 @@
         <version>1.3.0-SNAPSHOT</version>
     </parent>
     <artifactId>sort-sdk</artifactId>
-    <version>1.3.0-SNAPSHOT</version>
+    <version>${project.version}</version>
```

Comment:
This version definition is unnecessary, just remove it.

Reply:
OK